backend/activity_log/serializers.py

```diff
@@ -24,3 +21,23 @@ def get_date(self, instance):
 
     def get_total(self, instance):
         return instance["total"]
+
+
+class EventTypeSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = models.EventType
+        fields = ["name"]
+
+
+class LogEntrySerializer(serializers.ModelSerializer):
+    class Meta:
+        model = models.LogEntry
+        fields = ["id"]
+
+
+class ActivityLogEventTypeSerializer(serializers.Serializer):
```

Comment:
If this is the serialiser you are using why did you also create EventTypeSerializer and LogEntrySerilizer?